docs/builds/guides/development/dll-builds-collaboration-features.md

```diff
@@ -0,0 +1,176 @@
+---
+menu-title: DLL builds for CKEditor 5 Collaboration Features
+category: builds-development
+order: 40
+modified_at: 2022-02-21
+---
+
+# CKEditor 5 DLL builds for CKEditor 5 Collaboration Features
+
+<info-box>
+    This guide discusses using a DLL build together with CKEditor 5 Collaboration Features and is supplemental to the {@link builds/guides/development/dll-builds CKEditor 5 DLL builds guide}.
+
+    Make sure to read the {@link builds/guides/development/dll-builds base guide} first before proceeding.
+</info-box>
+
+## Anatomy of a DLL build with CKEditor 5 collaboration features
+
+A DLL build of the editor consists of the following parts:
+
+* **Base DLL build**. It is a single JavaScript file that combines the contents of several core CKEditor 5 packages: `utils`, `core`, `engine`, `ui`, `clipboard`, `enter`, `paragraph`, `select-all`, `typing`, `undo`, `upload`, and `widget`. These packages are either the framework core, or are features used by nearly all editor installations (`ckeditor5` on npm).
```

Comment:
```suggestion
* **Base DLL build**. It is a single JavaScript file that combines the contents of several core CKEditor 5 packages: `utils`, `core`, `engine`, `ui`, `clipboard`, `enter`, `paragraph`, `select-all`, `typing`, `undo`, `upload`, and `widget`. These packages are either the framework core, or are features used by nearly all editor installations. The build is available on NPM in `ckeditor5` package.
```

Reply:
How about wording it this way. The way it is worded now is not clear, the note "(`ckeditor5` on npm)" is sudden and is not connected with the previous sentence. If you agree with me (I might not understood something), please rewrite it in other places too.

Reply:
Or you can even leave it as it is in other places, at it will be now more clear what "(\`ckeditor5-collaboration\` on NPM)" means.